libs/core/src/lib/checkbox/checkbox/checkbox.component.ts

```diff
@@ -208,4 +208,11 @@ export class CheckboxComponent implements ControlValueAccessor {
     private _isSpaceBarEvent(event: KeyboardEvent): boolean {
         return event.keyCode === 32;
     }
+
+    /** Method to trigger change detection in component */
+    private _detectChanges(): void {
+        if (!this._changeDetectorRef['destroyed']) {
```

Comment:
I think if it doesnt exist we should detach it. This just hides the real cause. The error is a symptom not the cause.